README.md

```diff
@@ -27,6 +31,13 @@ This provider adds labels to all resources that are managed
 Also see the usage of vmpooler's optional purge_unconfigured_resources, which is used to delete any resource found that
 do not have the pool label, and can be configured to allow a specific list of unconfigured pool names. 
 
+### Pre-requisite
+
+- A service account needs to be created and a private json key generated (see usage section)
+- The service account needs given permissions to the project (broad permissions would be compute v1 admin and dns admin). A yaml file is provided that lists the least-privilege permissions needed
+- if using DNS, a DNS zone needs to be created
```

Comment:
Is this really optional? It kinda seems required based on the code

Reply:
I'll add some words, but yes if you dont specify the dns_zone_resource_name in the provider's config, the DNS setup/teardown will be skipped.